### DISCUSSION TITLE
✅ Arrow keys does not work after ssh login on iPhone

### DISCUSSION THREAD

**Discussion by sambana786** [isontheline/pro.webssh.net#1240] [Q&A] [answered]
How do you activate arrow keys to work on ssh connection iPhone latest iOS. All other keys insert etc all work. However the one on the center of the keyboard up arrow side arrow down Arrow I cannot get it to work in my connected session when editing files on terminal. Any feedback or assistance will be appreciated

**Comment by isontheline** [comment, marked as answer]
Hello @sambana786 👋

Just press and drag this component to the direction you want to go.

Could you make it a try?

Have a nice day ☀️

**Comment by sambana786** [comment]
Worked like a champ. Thank you so much for this solution. 

 

Sam

**Comment by isontheline** [comment]
You're welcome!
